Card.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CARD_H
+#define CARD_H
+class Card
+{
+public:
+  enum class  Rank
```

Comment:
nit: extra space between 'class' and 'Rank'. 

by 'nit' I mean this is a nitpick but still a worthwhile comment